src/schema/common.cljs

```diff
@@ -8,7 +8,20 @@
 (def ^:private ?customization-color
   [:or :string :keyword])
 
+(def ^:private ?public-key
+  [:re #"^0x04[0-9a-f]{128}$"])
+
+(def ^:private ?general-rpc-call
+  [:cat
```

Comment:
`cat` allows heterogeneous elements, but `:json-rpc/call` expects a sequence of one or more maps of the same structure.

So it's better if we use `:sequential`. We might as well add `{:min 1}` because it would be weird to dispatch an RPC effect with nothing to do.